File: examples/associative_array/associative_array.cpp
#include <iostream>
#include <map>
using namespace std;
int main() {
    map<string, string> monArray;
    monArray["1"] = "January";
    monArray["2"] = "February";
    monArray["3"] = "March";
    for (map<string, string>::iterator p = monArray.begin(); p != monArray.end(); ++p ) {
         cout << p->second << endl;
   }
}

File: examples/associative_array/AssociativeArray.java
import java.util.*;
public class AssociativeArray {
    public static void main(String[] args) {
        Map<String, String> map = new HashMap<String, String>();
        map.put("1", "January");
        map.put("2", "February");
        map.put("3", "March");
        System.out.println(map.get("1"));
        System.out.println(map.get("2"));
        System.out.println(map.get("3"));
    }
}
